packages/@internationalized/number/test/NumberParser.test.js

```diff
@@ -10,430 +10,1240 @@
  * governing permissions and limitations under the License.
  */
 
-import fc from 'fast-check';
-import messages from '../../../@react-aria/numberfield/intl/*.json';
-import {NumberParser} from '../src/NumberParser';
+import fc from "fast-check";
```

Comment:
looks like this file went through a prettier, you'll want to undo all of that otherwise it won't pass lint